odoo/fields.py

```diff
@@ -2433,6 +2433,12 @@ class Image(Binary):
     max_width = 0
     max_height = 0
     verify_resolution = True
+    def _setup_attrs(self, model_class, name):
+        super()._setup_attrs(model_class, name)
+        if not model_class._abstract:
+            assert model_class._auto or getattr(model_class, '_log_access', True), \
+                "Image field %s requires the model to have _log_access = True" % self
```

Comment:
This should be method `setup()` instead, and IMO you should use `warnings.warn()` instead of `assert`.  The latter will log once instead of "maybe crash"...

BTW, style-wise, functions or methods should be surrounded by one blank line.